src/NetAnalyzers/VisualBasic/Microsoft.NetCore.Analyzers/Runtime/BasicDetectPreviewFeatureAnalyzer.vb

```diff
@@ -11,23 +12,271 @@ Namespace Microsoft.NetCore.VisualBasic.Analyzers.Runtime
 
         Inherits DetectPreviewFeatureAnalyzer
 
+        Private Function IsSyntaxToken(identifier As SyntaxToken, previewInterfaceSymbol As ISymbol) As Boolean
+            Return identifier.ValueText.Equals(previewInterfaceSymbol.Name)
```

Comment:
Should be case insensitive?

Reply:
Good catch! And we'll need tests that verify that VB is handled without case sensitivity.